helpers/tests/unit/test_checkpoint_logger.py

```diff
@@ -497,11 +480,273 @@ def test_reliability_counters(self):
         ]
         with CounterAssertionSet(counter_assertions):
             checkpoints.log(DecoratedEnum.BRANCH_2_SUCCESS)
-        assert metrics.data["DecoratedEnum.events.BRANCH_2_SUCCESS"] == 1
-        assert metrics.data["DecoratedEnum.total.begun"] == 1
-        assert metrics.data["DecoratedEnum.total.failed"] == 1
-        assert metrics.data["DecoratedEnum.total.ended"] == 3
-        assert metrics.data["DecoratedEnum.total.succeeded"] == 2
+
+    @patch("helpers.checkpoint_logger.prometheus.CHECKPOINT_ENABLED_REPOSITORIES")
+    def test_reliability_counters_with_context(self, mock_object):
+        repoid = 123
+        mock_object.return_value = repoid
```

Comment:
`mock_object` is the feature object, not its `check_value()` function. i wonder why it appears to work anyway?

Reply:
Huh didn't post yesterday. I had a hard time mocking this but ultimately this did the trick, not 100% sure why